src/ui/popup/settings-page/index.tsx

```diff
@@ -16,6 +19,7 @@ export default function SettingsPage(props: SettingsPageProps) {
     return (
         <section class="m-section">
             <EnabledByDefaultGroup {...props} />
+            {isFirefox() ? <EnableCustomBrowserTheme {...props}/> : null}
```

Comment:
Hmm, I need to think where to put options like this, leave it here at this time.